src/data/priv_appendable_data.rs

```diff
@@ -323,17 +263,17 @@ impl PrivAppendableData {
 
 impl Debug for PrivAppendableData {
     fn fmt(&self, formatter: &mut Formatter) -> fmt::Result {
-        let previous_owner_keys: Vec<String> = self.previous_owner_keys
+        let previous_owner_keys: Vec<String> = self.signatures
             .iter()
-            .map(|pub_key| ::utils::format_binary_array(&pub_key.0))
+            .map(|(pub_key, _)| ::utils::format_binary_array(&pub_key.0))
             .collect();
-        let current_owner_keys: Vec<String> = self.current_owner_keys
+        let current_owner_keys: Vec<String> = self.owners
             .iter()
             .map(|pub_key| ::utils::format_binary_array(&pub_key.0))
             .collect();
-        let previous_owner_signatures: Vec<String> = self.previous_owner_signatures
+        let previous_owner_signatures: Vec<String> = self.signatures
             .iter()
-            .map(|signature| ::utils::format_binary_array(&signature.0[..]))
+            .map(|(_, sig)| ::utils::format_binary_array(&sig.0[..]))
```

Comment:
Please import `utils` (so that the `::` can be omitted) and update the debug output to reflect the new fields: It should say `owners:` instead of `current_owner_keys:` and just print the list of tuples from the map after `signatures:` instead of `previous_owner_keys:` and `previous_owner_signatures:`.